app-editors/kakoune/kakoune-2020.01.16_p20200601.ebuild

```diff
@@ -3,11 +3,13 @@
 
 EAPI=7
 
-MY_COMMIT="6fa26b8dd2ac0931fe688370728c47086277d883"
```

Comment:
what about that empty gcc-11 patch? :thinking:

Reply:
oops, added now